README.md

```diff
@@ -9,4 +9,158 @@ To install this project, you can simply run `sh install.sh` in your terminal. Th
 To run the scraper, you can use the run script by typing `sh run_scraper.sh` into your terminal. This will enable the virtual environment and run `scraper.py`. Once again, the virtual environment will not stay active after the script finishes running. If you want to run the scraper without the run script, enable the virtual environment, then run `python3 scraper.py`.
 
 ## Running the API
-The best way to run the API right now is to run the command `FLASK_APP="app.py" FLASK_ENV=development flask run;`. Note that this is not the best way to run the scraper at this time.
\ No newline at end of file
+The best way to run the API right now is to run the command `FLASK_APP="app.py" FLASK_ENV=development flask run;`. Note that this is not the best way to run the scraper at this time.
+
+## Data Model
```

Comment:
Do you want to merge the .data_scrapers/README  into the top level README? See PR #48

Reply:
@ldtcooper any thoughts on this point from @elaguerta? There are a lot of great improvements in #48 and it seems like we should have merged it quite a while ago.

This is kind of in conflict with that. I think we should merge #48. I’m not sure whether you’re thinking we should also move all that content to the top level README (seems pretty big and maybe a bit unwieldy, but that is not a strongly held opinion).

Reply:
So this now repeats a lot of information that is already in [`data_models/README.md`](https://github.com/sfbrigade/data-covid19-sfbayarea/tree/sonoma/data_models#age-group). Would it be better to just link to it? That way we don’t get in a situation where someone updates one place but not the other.

Reply:
I thought we were merging `data_models/README.md` into the main readme

Reply:
OK! I was unclear on exactly which direction we were going there. If we’re merging into the top-level README, then this is missing a bunch of stuff from `data_models/README.md`, and this PR should probably also remove that file.

Reply:
So… we’re just going to put this in here as-is?